ansible_collections/arista/avd/roles/eos_designs/templates/base/local-users.j2

```diff
@@ -4,10 +4,13 @@ local_users:
     privilege: {{ local_users[local_user].privilege }}
 {%     if local_users[local_user].role is defined %}
     role: {{ local_users[local_user].role }}
-{%      endif %}
-{%    if local_users[local_user].sha512_password is defined %}
+{%     endif %}
+{%     if local_users[local_user].sha512_password is defined %}
     sha512_password: {{ local_users[local_user].sha512_password }}
 {%     elif local_users[local_user].no_password is defined and local_users[local_user].no_password == True %}
```

Comment:
I know this is not your change, but since you are touching the file :)
Please change to `is arista.avd.defined(true)`